Update the extension details (#4)
OfficeDev/OfficeAddinDevKit: README.md

> ## What is Office Add-ins Development Kit?

Reviewer: Typo fixed by Copilot. Pls check.

### Create your add-in
Use the Office Add-ins Development Kit for Visual Studio Code to set up your first add-in project. Create your Office add-in project using the following steps:

* Ensure you've installed the Microsoft Office Add-ins Development Kit for Visual Studio Code.
* Select the Office Add-ins Development Kit icon in the Visual Studio Code **Activity Bar** to open the extension.
* Select **Create a New Add-in**.
* In the drop down that opens, select an Office app that you want to build the add-in for.
* Select an add-in template from the list of available templates.
* Select JavaScript/TypeScript as the programming language.
* In the **Workspace folder** dialog that opens, select the folder where you want to create the project.

> * Give a name to the project (with no spaces) when prompted. Office Add-ins Development Kit will create the project with basic files and scaffolding. It will then open the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.

Reviewer 2: Why no spaces?

> * Give a name to the project (with no spaces) when prompted. Office Add-ins Development Kit will create the project with basic files and scaffolding. It will then open the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.

Reviewer 2: I think it'll emphasize the behavior better if you italicize less of the line.
```suggestion
* Give a name to the project (with no spaces) when prompted. Office Add-ins Development Kit will create the project with basic files and scaffolding. It will then open the project in a *new* Visual Studio Code window. You are free to close the original Visual Studio Code window.

```

> - **[Node.js](https://nodejs.org) 16, 18, or 20 (18 is preferred) and [npm](https://www.npmjs.com/get-npm).** To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.

Reviewer: Node should be installed by developer themselves. We will run a checker in Dev Kit to inform them if Node.js is not installed.

When a add-in project is created, you can explore and customize the components by reviewing the key files listed below.

- The `./manifest.xml` file in the root directory of the project defines the settings and capabilities of the add-in.  <br>You can check whether your manifest file is valid by selecting **Validate Manifest File** option from the Office Add-ins Development Kit.
- The `./src/taskpane/taskpane.html` file contains the HTML markup for the task pane.
- The `./src/taskpane/taskpane.css` file contains the CSS that's applied to content in the task pane.
- The `./src/taskpane/taskpane.js` file contains the Office JavaScript API code that facilitates interaction between the task pane and the Office application.

### Preview your add-in in Office apps

To understand how the add-in will work in Office apps, you can use the Office Add-ins Development Kit to easily run and debug your Office add-in in your local dev environment.

> <br><img src=./assets/Dev_Kit_Preview.gif alt="Preview an add-in using Office Add-ins Development Kit"/>

Reviewer 2: The alt text needs to fully describe what is happening to a person who can't see the GIF.
```suggestion
<br><img src=./assets/Dev_Kit_Preview.gif alt="A user selecting the 'Preview' option from the Office Add-ins Development Kit task pane, then choosing the 'Edge Desktop' option to run the sample."/>

```

#### Preview Your Office Add-in (F5)

> Select **Preview Your Office Add-in(F5)** to launch the add-in and debug the code. In the drop down menu, select the option **Desktop (Edge Chromium)**.

Reviewer 2: "dropdown" is one word in this context. Also, if you can refer to the actual name of this menu, instead of the control type, that would be better.

Also, the option seems to be named "Edge Desktop (Edge Chromium)".
```suggestion
Select **Preview Your Office Add-in(F5)** to launch the add-in and debug the code. In the dropdown menu, select the option **Edge Desktop (Edge Chromium)**.

```

> The extension then checks that the prerequisites are met before debugging starts. Check the terminal for detailed information if there are issues with your environment. After this process, the Office desktop application launches and sideloads the add-in.

Reviewer 2: Avoid repeating the word "check".
```suggestion
The extension then checks that the prerequisites are met before debugging starts. The terminal will display detailed information if there are issues with your environment. After this process, the Office desktop application launches and sideloads the add-in.
```

> You can also start debugging by hitting the `F5` key or running `npm run start` command in the terminal.

Reviewer 2: Avoid the word "hitting" in favor of "pressing". It will translate better.
```suggestion
You can also start debugging by pressing the `F5` key or running the `npm run start` command in the terminal.
```
